docs/outputs.rst

```diff
@@ -150,4 +150,78 @@ I012   ignored            ign_add1
 
 Visual reports
 --------------
-We're working on it.
+Static visual reports can be generated by using the ``--png`` flag when calling
+tedana from the command line.
+Images are created and placed within the output directory, in a folder labeled
+``figures``.
+
+These reports consist of three main types of images.
+
+Component Images
+````````````
+.. image:: /_static/example_good_component.png
+  :align: center
+
+For each component identified by tedana, a single image will be created.
+Above is an example of an accepted component.
+These are designed for an up-close inspection of both the spatial and temporal
+aspects of the component, as well as ancillary information.
+
+The title of the plot provides information about variance, kappa and rho values
+as well as the reasons for rejection, if any (see above for codes).
+
+Below this is the component timeseries, color coded on the basis of its
+classification.
+Green for accepted, Red for rejected, Black for ignored or unclassified.
+
+Slices are then selected from sagittal, axial and coronal planes, to highlight
+the component pattern.
+By default these images used the red-blue colormap and are scaled to 10% of the
+max beta value.
+
+.. note::
+  You can select your own colormap to use by specifying its name when calling
+  tedana with ``--png-cmap``.
+  For example, to use the bone colormap, you would add ``--png-cmap bone``.
+
+Finally, the bottom of the image shows the Fast Fourier Transform of the
+component timeseries.
+
+Tip: Look for your fundamental task frequencies here!
+
+
+.. image:: /_static/example_bad_component.png
+  :align: center
+
+Above, you can appreciate a component that was rejected.
```

Comment:
```suggestion
Above, you can review a component that was rejected.
```

Reply:
I think that "appreciate" is just slightly too vague. I think I'd meant to put it in my review, but I don't feel strongly that you should change it; it's a matter of preference.